Gaijinizer patch/Map404.txt

```diff
@@ -1,238 +1,296 @@
+fully understand without even binding them with
+a promise first...#Ron
 > END STRING
 
 > BEGIN STRING
 いくらスフィンクスがアホバカマヌケで頼りないからって
 なにもこんなやつの能力に頼らなくても……
 
 そのまま裏切られたらどうするつもりだったんだよ！？
-> CONTEXT: Map404/events/1/pages/0/36/Dialogue/other1 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/36/Dialogue/other1
+No matter how much of a complete airhead moron
+Sphinx might be, it's infinitely better to rely
+on her than this person's abilities. What was
+your plan if she'd betrayed you?!#Ron
 > END STRING
 
 > BEGIN STRING
 い、いや……
 べつにスフィンクスが頼りなかったわけじゃないよ
 
 それに、賭けに出てたってワケでもないんだ。
-> CONTEXT: Map404/events/1/pages/0/42/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/42/Dialogue/kazuya
+N-No, it's not like I wasn't going to rely on
+Sphinx at all. And it's not like this plan was
+some big gamble, either.#Ron
 > END STRING
 
 > BEGIN STRING
 彼女は、『必ず』僕を解放する。
 
 ちゃんと確信があったから、この話に乗ったんだよ！
-> CONTEXT: Map404/events/1/pages/0/47/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/47/Dialogue/kazuya
+I went along with her plan because she'd
+"definitely" be able to change me back to normal!#Ron
 > END STRING
 
 > BEGIN STRING
 ……………
 
 どうしてそう思ったの？
-> CONTEXT: Map404/events/1/pages/0/52/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map404/events/1/pages/0/52/Dialogue/enemy29
+......
 
+What made you think that?#Ron
 > END STRING
 
 > BEGIN STRING
 さっき君は、自分で言っていたじゃないか。
 『一方的な戦いは、見てて気分悪くなる』って
 
 わざわざ、練気の刀を渡したりするほど……
-> CONTEXT: Map404/events/1/pages/0/59/Dialogue/kazuya-d < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/59/Dialogue/kazuya-d
+You told me before that you don't like to see
+one-sided battles, right? So much so that you went
+out of your way to give me my shrunken sword.#Ron
 > END STRING
 
 > BEGIN STRING
 僕を最初に襲ったときもそうだよ。
 
 『キミに選択肢がないと、フェアじゃない』
 拘束したとき、君はハッキリそう言った。
-> CONTEXT: Map404/events/1/pages/0/64/Dialogue/kazuya-d < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/64/Dialogue/kazuya-d
+It was the same when you attacked me that first
+time. When you had me bound, you clearly said it
+wouldn't be fair if you didn't give me a choice.#Ron
 > END STRING
 
 > BEGIN STRING
 君は、\\C[2]『一方的な勝利』をとことん嫌ってる。\\C[0]
 
 だから、妖精サイズで、ましてや丸裸の相手を
 一方的に蹂躙するなんてあり得ないんだ。
-> CONTEXT: Map404/events/1/pages/0/69/Dialogue/kazuya-d < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/69/Dialogue/kazuya-d
+\\C[2]You detest one-sided victories.\\C[0]
+So seemed pretty unlikely that you'd one-sidedly
+take advantage to trample a fairy-sized opponent,
+let alone one who's stark naked.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 それに、僕をどうにかするために行動するなら
 そもそも声なんかかけないし、捕まえるのも容易なはずだ
 
 それをしなかった時点で、君に敵意はない。
-> CONTEXT: Map404/events/1/pages/0/74/Dialogue/kazuya-d < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/74/Dialogue/kazuya-d
+If you were going to do something to me, you
+wouldn't have called out to me first, and
+capturing me would've been simple. Those things
+told me you hadn't come with hostile intent.#Ron
 > END STRING
```

Comment:
Minor revisions:
```
If you were going to do something to me, you
would have just captured me without saying a
word instead of approaching. Those things
told me you hadn't come with hostile intent.
```